### DISCUSSION TITLE
Blogs list in alphabetical order

### DISCUSSION THREAD

**Discussion by martienne17** [AprilSylph/XKit-Rewritten#1246] [Suggestion Box]
I know this is 90% only a problem for me because I have a shit memory but I would love it if you could make it so the blogs you run list in alphabetical order instead of whatever random-ass order it displays in currently (I know it's the order of creation but I can never remember which order the list is in and I _always_ have to hunt for the blog I want to open).

**Comment by marcustyphoon** [comment]
You can rearrange your sideblogs in Tumblr itself via click-and drag! Buuuuuuut only on desktop web, and (for some reason that I don't understand) only if Tumblr doesn't detect your device as having a touchscreen and switch to a view with slightly larger text.

Do you see these set-of-four-squares icons in your Tumblr sidebar?

<img width="428" src="https://github.com/AprilSylph/XKit-Rewritten/assets/8336245/7b031230-f22f-40af-a32d-1a8ae78960d9">

**Reply by AprilSylph** [reply]
> Buuuuuuut only on desktop web, and (for some reason that I don't understand) only if Tumblr doesn't detect your device as having a touchscreen

The library that Tumblr uses for draggable components is old and not touch-friendly. If you simulate a touchscreen and modify the CSS to show drag handles, about half the time you drag blogs it'll drag and scroll at the same time.

**Reply by marcustyphoon** [reply]
Ooh, fun.

**Comment by martienne17** [comment]
Well dang

Thanks for letting me know, I never noticed those before

**Comment by marcustyphoon** [comment]
They are indeed anything but obvious in what they're supposed to actually do :D
